test/app_test_case.py

```diff
@@ -38,6 +38,8 @@ def __init__(self, app, config, host, port):
         self.server_wrapper = LocalDevServer(app, config, host, port)
 
     def run(self):
+        log.debug(f'Serving on http://{self.address[0]}:{self.address[1]}')
```

Comment:
Did you test your changes? the `log` has it's level set to `INFO`. There's no output in my PyCharm instance.

Your changes: 
```
> (cd test && python -m unittest service.test_response)
2021-11-18 11:29:04,006 DEBUG   MainThread: Successfully wrote aggregate for test/files/c18f9014-7086-5216-a73a-e37af51f2bbb with 1 contribution(s).
2021-11-18 11:29:04,973 INFO    Thread-2: Received GET request for '/health/basic', with query null and headers {"host": "127.0.0.1:59904", "user-agent": "python-requests/2.26.0", "accept-encoding": "gzip, deflate, br", "accept": "*/*", "connection": "keep-alive"}.
2021-11-18 11:29:04,973 INFO    Thread-2: Did not authenticate request.
2021-11-18 11:29:04,973 DEBUG   Thread-2: Returning 200 response with headers {"Access-Control-Allow-Origin": "*", "Access-Control-Allow-Headers": "Authorization,Content-Type,X-Amz-Date,X-Amz-Security-Token,X-Api-Key"}. See next line for the first 1024 characters of the body.
{"up": true}
127.0.0.1 - - [18/Nov/2021 11:29:04] "GET /health/basic HTTP/1.1" 200 -
```
INFO
```
> (cd test && python -m unittest service.test_response)
2021-11-18 11:27:11,896 DEBUG   MainThread: Successfully wrote aggregate for test/files/c18f9014-7086-5216-a73a-e37af51f2bbb with 1 contribution(s).
2021-11-18 11:27:12,827 INFO    Thread-1: Serving on http://127.0.0.1:59860
2021-11-18 11:27:12,832 INFO    Thread-2: Received GET request for '/health/basic', with query null and headers {"host": "127.0.0.1:59860", "user-agent": "python-requests/2.26.0", "accept-encoding": "gzip, deflate, br", "accept": "*/*", "connection": "keep-alive"}.
```